src/main/java/techItEasy/techItEasy/Controllers/TelevisionsController.java

```diff
@@ -0,0 +1,54 @@
+package techItEasy.techItEasy.Controllers;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import techItEasy.techItEasy.Models.Televisions;
+import techItEasy.techItEasy.Repositories.TelevisionsRepository;
+
+import java.util.Optional;
+
+@RestController
+public class TelevisionsController {
+
+@Autowired
+    TelevisionsRepository televisionsRepository;
+
```

Comment:
Je hebt je @Mappings allemaal benoemd en (bijna) de juiste paden er naar toe gegeven. Ook heb je de juiste returns gegeven. 
Daartussen moet je je data ook nog uit je repository ophalen. Zie de uitwerkingen voor hoe zij dat gedaan hebben. Het is nog wat meer werk dus.
Verder kan je in de tweede regel ipv een ResponseEntity<Object> deze nu verder specificeren tot een ResponsEntity<Television>